cli/integration-test/integration/helper.clj

```diff
@@ -50,6 +50,22 @@
        (doseq [[r# m#] (map vector res# maps#)]
          (assert-submap m# r#)))))
 
+(defmacro assert-contains-submaps
+  "Asserts that maps are contained submaps of result in results. "
+  [maps result]
+  `(let [maps# ~maps
+         res# ~result]
+     (doseq [m# maps#]
+       (let [m-keys (keys m#)]
+         (is (some #(= m# (some-> % (select-keys m-keys))) res#)))))) 
+(comment
+  (let [maps# [{:a 1 :b 1} {:a 3}]
+        res# [{:a 1 :b 1} {:a 3 :b 5}]]
+    (doseq [m# maps#]
+      (let [m-keys (keys m#)]
+        (is (some #(= m# (some-> % (select-keys m-keys))) res#)))))
+  )
+
```

Comment:
I'm not a big fan of comments on the middle of the file, can we remove it?

Reply:
Yes, I'll remove it.
Do you like them at the end or not at all?

Reply:
I accept then on the end, but personal I remove it, but feel free to choose